api-spec/protobuf/ark/v1/explorer.proto

```diff
@@ -21,6 +21,11 @@ service ExplorerService {
       get: "/v1/vtxos/{address}"
     };
   };
+  rpc SubscribeForAddress(SubscribeForAddressRequest) returns (stream SubscribeForAddressResponse) {
```

Comment:
It means 1 stream / address, should we allow the client to subscribe several addresses in the same RPC ?

Reply:
Currently we support single key wallets in our clients, don't think is necessary at the moment?

Reply:
Yes, but in theory fulmine has also to call this method for VHTLC?

Reply:
They are commit-reveal type of contract, you subscribe once, wait for being "spent" and then you unsubscribe. 
Rarely you fund multiple contracts at same time. I'm not against it, just wondering if is worth as we don't have strong need for now

Reply:
We can always extend via a new rpc later `SubscribeForAddresses`